src/test/kotlin/edu/carroll/cs389/service/tracker/trackservice/TrackServiceQueryTest.kt

```diff
@@ -0,0 +1,561 @@
+package edu.carroll.cs389.service.tracker.trackservice
+
+import edu.carroll.cs389.service.tracker.TrackService
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.annotation.DirtiesContext
+import org.springframework.test.util.AssertionErrors.assertTrue
+import java.sql.Timestamp
+import java.util.Date
+
+@SpringBootTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
```

Comment:
I *think* you could use @Transactional instead of the more complex @DirtiesContext annotation.

[ Not, the `Transactional` annotation gets rewritten by Github  to lowercase text due to how it does references of users).